data_structures/src/radon_error.rs

```diff
@@ -57,6 +57,16 @@ pub enum RadonErrors {
     ArrayIndexOutOfBounds = 0x70,
     /// Tried to access a value from a map using a key that does not exist
     MapKeyNotFound = 0x71,
+    // Bridge errors: errors that only belong in inter-client communication
+    /// Requests that cannot be parsed must always get this error as their result.
+    /// However, this is not a valid result in a Tally transaction, because invalid requests
+    /// are never included into blocks and therefore never get a Tally in response.
+    BridgeMalformedRequest = 0xE0,
+    /// Witnesses exceeds 100
+    BridgeTooManyWitnesses = 0xE1,
+    /// The request is rejected on the grounds that it may cause the submitter to spend or stake an
+    /// amount of value that is unjustifiably high when compared with the reward they will be getting
+    BridgePoorIncentives = 0xE2,
```

Comment:
We could use input from @tomsanch on whether _poor_ or _weak_ sounds better:
```suggestion
    BridgeWeakIncentives = 0xE2,
```

Reply:
I would go for PoorIncentives

Reply:
Thanks @tomsanch!